src/models/vehicle/Vehicle.tsx

```diff
@@ -1,26 +1,23 @@
 import { MathUtils, PerspectiveCamera, Vector3 } from 'three'
-import { useEffect, useLayoutEffect, useRef } from 'react'
+import { useLayoutEffect } from 'react'
 import { useFrame, useThree } from '@react-three/fiber'
-import { PositionalAudio } from '@react-three/drei'
 import { useRaycastVehicle } from '@react-three/cannon'
 
 import type { PropsWithChildren } from 'react'
-import type { WheelInfoOptions } from '@react-three/cannon'
-import type { PositionalAudio as PositionalAudioImpl } from 'three'
+import type { BoxProps, WheelInfoOptions } from '@react-three/cannon'
 
-import { Dust, Skid, Boost } from '../../effects'
+import { AccelerateAudio, BoostAudio, Boost, BrakeAudio, Dust, EngineAudio, HonkAudio, Skid } from '../../effects'
 import { getState, mutation, useStore } from '../../store'
 import { useToggle } from '../../useToggle'
 import { Chassis } from './Chassis'
 import { Wheel } from './Wheel'
 
 import type { Controls, WheelInfo } from '../../store'
-import type { ChassisProps } from './Chassis'
 
 const { lerp } = MathUtils
 const v = new Vector3()
 
-type VehicleProps = PropsWithChildren<Pick<ChassisProps, 'angularVelocity' | 'position' | 'rotation'>>
+type VehicleProps = PropsWithChildren<Pick<BoxProps, 'angularVelocity' | 'position' | 'rotation'>>
```

Comment:
Not sure why this was changed?

Reply:
ChassisProps was just BoxProps, so no need for it.